build.gradle

```diff
@@ -249,37 +243,55 @@ dependencies {
-        }
+        implementation 'bio.terra:terra-common-lib:0.1.10-SNAPSHOT'
     }
 
-    implementation 'bio.terra:terra-policy-client:1.0.4-SNAPSHOT'
-    implementation 'bio.terra:terra-resource-buffer-client:0.4.3-SNAPSHOT'
-    implementation 'bio.terra:externalcreds-client-resttemplate:0.72.0-SNAPSHOT'
-    implementation 'org.glassfish.jersey.inject:jersey-hk2:2.30.1'
+    implementation 'org.broadinstitute.dsde.workbench:sam-client_2.13:0.1-a91acd0'
+    implementation 'bio.terra:terra-policy-client:1.0.11-SNAPSHOT'
+    implementation 'bio.terra:terra-resource-buffer-client:0.198.42-SNAPSHOT'
+    implementation 'bio.terra:externalcreds-client-resttemplate:1.3.0-SNAPSHOT'
 
-    // Pin the version of the OkHttp client to avoid a conflict with the version used by the Sam client
-    implementation 'com.squareup.okhttp3:okhttp:4.10.0'
+    implementation 'org.glassfish.jersey.inject:jersey-hk2'
+
+    implementation 'com.squareup.okhttp3:okhttp'
     implementation 'org.springframework.boot:spring-boot-starter-actuator'
     implementation 'io.micrometer:micrometer-registry-prometheus'
 
-    implementation 'com.fasterxml.jackson.core:jackson-core:2.13.2'
-    implementation 'com.fasterxml.jackson.core:jackson-databind:2.13.2'
+    implementation 'com.fasterxml.jackson.core:jackson-core:2.15.3'
+    implementation 'com.fasterxml.jackson.core:jackson-annotations:2.15.3'
+    implementation 'com.fasterxml.jackson.core:jackson-databind:2.15.3'
 
     // Azure related dependencies
-    implementation 'com.azure:azure-identity:1.9.1'
-    implementation 'com.azure.resourcemanager:azure-resourcemanager:2.28.0'
-    implementation 'com.azure.resourcemanager:azure-resourcemanager-loganalytics:1.0.0-beta.3'
+    implementation 'com.azure:azure-identity:1.11.1'
+    implementation 'com.azure.resourcemanager:azure-resourcemanager:2.34.0'
+    implementation 'com.azure.resourcemanager:azure-resourcemanager-loganalytics:1.0.0'
     implementation 'com.azure.resourcemanager:azure-resourcemanager-securityinsights:1.0.0-beta.4'
-    implementation 'com.azure:azure-storage-common:12.21.1'
-    implementation 'com.azure:azure-storage-file-datalake:12.15.2'
-    implementation 'com.azure:azure-data-tables:12.3.11'
+    implementation 'com.azure:azure-storage-common:12.24.1'
+    implementation 'com.azure:azure-storage-file-datalake:12.18.1'
+    implementation 'com.azure:azure-data-tables:12.3.18'
 
     implementation platform('io.sentry:sentry-bom:6.25.0') //import bom
     implementation('io.sentry:sentry-spring-boot-starter')
     implementation('io.sentry:sentry-logback')
 
+    // OpenTelemetry
+    var openTelemetryVersion = '1.32.0'
+    implementation "io.opentelemetry:opentelemetry-api:${openTelemetryVersion}"
+    implementation "io.opentelemetry:opentelemetry-sdk:${openTelemetryVersion}"
+    //implementation "io.opentelemetry:opentelemetry-sdk-common:${openTelemetryVersion}"
```

Comment:
Do we want to keep this in?

Reply:
I've kept it hanging around since I think it was added with one of the other service's integration with otel, but it breaks the build! So, I'll remove it!